### PR TITLE
Milestone #1: Basic layout

### PR DESCRIPTION
**Pull request by GuilhermeGiachellin**
## What was made in this project 

-Added basic layout of the page.
-Html, CSS, and Js were used.

### CODE REVIEW

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @GuilhermeGiachellin  👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
#### Highlights
- good GitHub workflow ✔️ 
- not forming any linter errors ✔️ 
- good job on the js files

## Required Changes ♻️

- [x] Kindly remove `dist` folder from gitiginore. This folder will help you in the deployment of your app at a later stage

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

File: src/style.css [hunk 1]
```diff
@@ -4,6 +4,104 @@
   list-style: none;
 }
```
Comment (carlos-ssh) [inline src/style.css:6]: Kindly enable the correct linters for your project.
- Keep your general requirement's setup using the [linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) described.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @GuilhermeGiachellin, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abruzy — APPROVED** [review]
```markdown
Hi @GuilhermeGiachellin 👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**